firebase.json

```diff
@@ -14,6 +14,11 @@
           "source": "/hardware/:arch/quickstart{,/**}",
           "destination": "/hardware/:arch/zephyr-quickstart",
           "type": 301
+        },
+        {
+          "source": "/cloud/tiers",
+          "destination": "https://golioth.io/pricing",
+          "type": 301
```

Comment:
@hasheddan is this a reasonable way to add 301 redirect for this page, or should we do these on the firebase web console?